modules/utils/keyboard_util.py

```diff
@@ -96,7 +96,11 @@ def get_vote_kb(published_post: PublishedPost = None) -> InlineKeyboardMarkup:
                                                 callback_data=f"meme_vote,{reaction_id}"))
         keyboard.append(new_row)
     # the last button in the last row will be the report button
-    keyboard[-1].append(InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,"))
+    if config_map['meme']['report']:
+        if len(keyboard) > 0:
+            keyboard[-1].append(InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,"))
+        else:
+            keyboard.append([InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,")])
```

Comment:
I would like to suggest to put in a REPORT variable the content `InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,")` outside this if/else and use here the REPORT variable to reduce redundancy code and improve readability
ex.
```python
       REPORT = InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot,")
       if len(keyboard):
            keyboard[-1].append(REPORT)
        else:
            keyboard.append([REPORT])
```

Reply:
(why there is the comma `,` at the end of meme_report_spot?